_docs/style/spacing.md

```diff
@@ -60,16 +60,18 @@ Some cases may require different spacing to accommodate very small or large cont
 
 * **Small = 12px** 
 * **Medium = 20px**
-* **Large = 28px**
+* **Large = 32px**
+* **XL = 48px**
```

Comment:
s/`XL`/`Extra-Large`
s/`2XL`/`Extra-extra-large`
(Maybe it's just me, though. I'd want to see them spelled out because we spelled out the others.)

Reply:
Would we want to name similar to the toolkit? 

<img width="301" alt="screen shot 2018-04-19 at 7 41 44 pm" src="https://user-images.githubusercontent.com/545605/39024972-be8fce56-4409-11e8-9927-a024e9ae3445.png">

Reply:
How about somewhere in the middle?

Small (SM) = 12px
...
Extra-large (XL) = 48px